dataprofiler/tests/data/csv/daily-activity-sheet-tab.csv

```diff
@@ -0,0 +1,31 @@
+Time	Activity 	Supplies	Notes

```

Comment:
what if we have the line `COVID19 Activity Sheet` as in the other file?

Reply:
That's not a problem, I just left it this way to try various things (tabs in this case)